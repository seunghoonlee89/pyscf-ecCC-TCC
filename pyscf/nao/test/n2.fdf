SystemName          Nitrogen dimer 
SystemLabel         n2

%block ChemicalSpeciesLabel
 1  7  N.gga
%endblock ChemicalSpeciesLabel

AtomicCoordinatesFormat  Ang
AtomCoorFormatOut Ang

NumberOfAtoms       2
NumberOfSpecies     1

%block AtomicCoordinatesAndAtomicSpecies
    0.55843945    0.63860400    0.00000000   1       1  N.gga
   -0.55843944    0.63860400    0.00000000   1       2  N.gga
%endblock AtomicCoordinatesAndAtomicSpecies

### Molecular dynamics 
ElectronicTemperature      300 K
MD.TypeOfRun               CG
MD.NumCGsteps              0
MaxSCFIterations           100

COOP.Write     .true.
WriteDenchar   .true.
xml.write      .true.
