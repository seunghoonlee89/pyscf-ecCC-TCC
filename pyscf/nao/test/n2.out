Siesta Version: siesta-4.0--530
Architecture  : unknown
Compiler flags: mpif90 -O2 -fPIC -ftree-vectorize -march=native
PP flags      : -DFC_HAVE_ABORT 
SERIAL version

* Running in serial mode
>> Start of run:   5-DEC-2017  10:12:55

                           ***********************       
                           *  WELCOME TO SIESTA  *       
                           ***********************       

reinit: Reading from standard input
************************** Dump of input data file ****************************
SystemName          Nitrogen dimer
SystemLabel         n2
%block ChemicalSpeciesLabel
 1  7  N.gga
%endblock ChemicalSpeciesLabel
AtomicCoordinatesFormat  Ang
AtomCoorFormatOut Ang
NumberOfAtoms       2
NumberOfSpecies     1
%block AtomicCoordinatesAndAtomicSpecies
    0.55843945    0.63860400    0.00000000   1       1  N.gga
   -0.55843944    0.63860400    0.00000000   1       2  N.gga
%endblock AtomicCoordinatesAndAtomicSpecies
### Molecular dynamics
ElectronicTemperature      300 K
MD.TypeOfRun               CG
MD.NumCGsteps              0
MaxSCFIterations           100
COOP.Write     .true.
WriteDenchar   .true.
xml.write      .true.
************************** End of input data file *****************************

reinit: -----------------------------------------------------------------------
reinit: System Name: Nitrogen dimer
reinit: -----------------------------------------------------------------------
reinit: System Label: n2                                                          
reinit: -----------------------------------------------------------------------

initatom: Reading input for the pseudopotentials and atomic orbitals ----------
 Species number:            1  Label: N.gga Atomic number:           7
Ground state valence configuration:   2s02  2p03
Reading pseudopotential information in formatted form from N.gga.psf

Valence configuration for pseudopotential generation:
2s( 2.00) rc: 1.48
2p( 3.00) rc: 1.48
3d( 0.00) rc: 1.48
4f( 0.00) rc: 1.48
For N.gga, standard SIESTA heuristics set lmxkb to 3
 (one more than the basis l, including polarization orbitals).
Use PS.lmax or PS.KBprojectors blocks to override.

<basis_specs>
===============================================================================
N.gga                Z=   7    Mass=  14.010        Charge= 0.17977+309
Lmxo=1 Lmxkb= 3    BasisType=split      Semic=F
L=0  Nsemic=0  Cnfigmx=2
          n=1  nzeta=2  polorb=0
            splnorm:   0.15000    
               vcte:    0.0000    
               rinn:    0.0000    
               qcoe:    0.0000    
               qyuk:    0.0000    
               qwid:   0.10000E-01
                rcs:    0.0000      0.0000    
            lambdas:    1.0000      1.0000    
L=1  Nsemic=0  Cnfigmx=2
          n=1  nzeta=2  polorb=1
            splnorm:   0.15000    
               vcte:    0.0000    
               rinn:    0.0000    
               qcoe:    0.0000    
               qyuk:    0.0000    
               qwid:   0.10000E-01
                rcs:    0.0000      0.0000    
            lambdas:    1.0000      1.0000    
-------------------------------------------------------------------------------
L=0  Nkbl=1  erefs: 0.17977+309
L=1  Nkbl=1  erefs: 0.17977+309
L=2  Nkbl=1  erefs: 0.17977+309
L=3  Nkbl=1  erefs: 0.17977+309
===============================================================================
</basis_specs>

atom: Called for N.gga                 (Z =   7)

read_vps: Pseudopotential generation method:
read_vps: ATM3      Troullier-Martins                       
Total valence charge:    5.00000

read_vps: Pseudopotential includes a core correction:
read_vps: Pseudo-core for xc-correction

xc_check: Exchange-correlation functional:
xc_check: Ceperley-Alder
xc_check: WARNING: Pseudopotential generated with GGA PBE functional
V l=0 = -2*Zval/r beyond r=  1.4606
V l=1 = -2*Zval/r beyond r=  1.4606
V l=2 = -2*Zval/r beyond r=  1.4606
V l=3 = -2*Zval/r beyond r=  1.4606
All V_l potentials equal beyond r=  1.4606
This should be close to max(r_c) in ps generation
All pots = -2*Zval/r beyond r=  1.4606

VLOCAL1: 99.0% of the norm of Vloc inside     20.347 Ry
VLOCAL1: 99.9% of the norm of Vloc inside     46.372 Ry
atom: Maximum radius for 4*pi*r*r*local-pseudopot. charge    1.76189
atom: Maximum radius for r*vlocal+2*Zval:    1.53550
GHOST: No ghost state for L =  0
GHOST: No ghost state for L =  1
GHOST: No ghost state for L =  2
GHOST: No ghost state for L =  3

KBgen: Kleinman-Bylander projectors: 
   l= 0   rc=  1.634559   el= -1.348537   Ekb=  5.297520   kbcos=  0.333054
   l= 1   rc=  1.614250   el= -0.512415   Ekb= -5.262296   kbcos= -0.428933
   l= 2   rc=  1.718383   el=  0.002346   Ekb= -1.271998   kbcos= -0.007619
   l= 3   rc=  1.761893   el=  0.003449   Ekb= -0.494630   kbcos= -0.001017

KBgen: Total number of  Kleinman-Bylander projectors:   16
atom: -------------------------------------------------------------------------

atom: SANKEY-TYPE ORBITALS:
atom: Selected multiple-zeta basis: split     

SPLIT: Orbitals with angular momentum L= 0

SPLIT: Basis orbitals for state 2s

SPLIT: PAO cut-off radius determined from an
SPLIT: energy shift=  0.020000 Ry

   izeta = 1
                 lambda =    1.000000
                     rc =    3.683980
                 energy =   -1.330014
                kinetic =    1.203086
    potential(screened) =   -2.533100
       potential(ionic) =   -8.093205

   izeta = 2
                 rmatch =    2.905100
              splitnorm =    0.150000
                 energy =   -1.134717
                kinetic =    1.810562
    potential(screened) =   -2.945280
       potential(ionic) =   -8.899916

SPLIT: Orbitals with angular momentum L= 1

SPLIT: Basis orbitals for state 2p

SPLIT: PAO cut-off radius determined from an
SPLIT: energy shift=  0.020000 Ry

   izeta = 1
                 lambda =    1.000000
                     rc =    4.388595
                 energy =   -0.493943
                kinetic =    3.501321
    potential(screened) =   -3.995264
       potential(ionic) =   -9.374806

   izeta = 2
                 rmatch =    2.978652
              splitnorm =    0.150000
                 energy =   -0.289183
                kinetic =    5.180785
    potential(screened) =   -5.469968
       potential(ionic) =  -11.438899

POLgen: Perturbative polarization orbital with L=  2

POLgen: Polarization orbital for state 2p

   izeta = 1
                     rc =    4.388595
                 energy =    1.759924
                kinetic =    3.542120
    potential(screened) =   -1.782196
       potential(ionic) =   -6.226799
atom: Total number of Sankey-type orbitals: 13

atm_pop: Valence configuration (for local Pseudopot. screening):
 2s( 2.00)                                                            
 2p( 3.00)                                                            
Vna: chval, zval:    5.00000   5.00000

Vna:  Cut-off radius for the neutral-atom potential:   4.388595
comcore: Pseudo-core radius Rcore=  1.554824

atom: _________________________________________________________________________

prinput: Basis input ----------------------------------------------------------

PAO.BasisType split     

%block ChemicalSpeciesLabel
    1    7 N.gga                   # Species index, atomic number, species label
%endblock ChemicalSpeciesLabel

%block PAO.Basis                 # Define Basis set
N.gga                 2                    # Species label, number of l-shells
 n=2   0   2                         # n, l, Nzeta 
   3.684      2.905   
   1.000      1.000   
 n=2   1   2 P   1                   # n, l, Nzeta, Polarization, NzetaPol
   4.389      2.979   
   1.000      1.000   
%endblock PAO.Basis

prinput: ----------------------------------------------------------------------

coor:   Atomic-coordinates input format  =     Cartesian coordinates
coor:                                          (in Angstroms)

siesta: Atomic coordinates (Bohr) and species
siesta:      1.05530   1.20679   0.00000  1        1
siesta:     -1.05530   1.20679   0.00000  1        2

siesta: Automatic unit cell vectors (Ang):
siesta:    6.337723    0.000000    0.000000
siesta:    0.000000    5.109156    0.000000
siesta:    0.000000    0.000000    5.109156

siesta: System type = molecule  

initatomlists: Number of atoms, orbitals, and projectors:      2    26    32

siesta: ******************** Simulation parameters ****************************
siesta:
siesta: The following are some of the parameters of the simulation.
siesta: A complete list of the parameters used, including default values,
siesta: can be found in file out.fdf
siesta:
redata: Non-Collinear-spin run           =     F
redata: SpinPolarized (Up/Down) run      =     F
redata: Number of spin components        =     1
redata: Long output                      =     F
redata: Number of Atomic Species         =        1
redata: Charge density info will appear in .RHO file
redata: Write Mulliken Pop.              =     NO
redata: Mesh Cutoff                      =   100.0000  Ry
redata: Net charge of the system         =     0.0000 |e|
redata: Min. number of SCF Iter          =        0
redata: Max. number of SCF Iter          =      100
redata: Mix DM or H after convergence    =     F
redata: Recompute H after scf cycle      =     F
redata: Mixing is linear
redata: Mix DM in first SCF step ?       =     F
redata: Write Pulay info on disk?        =     F
redata: Discard 1st Pulay DM after  kick =     F
redata: New DM Mixing Weight             =     0.2500
redata: New DM Occupancy tolerance       = 0.000000000001
redata: No kicks to SCF
redata: DM Mixing Weight for Kicks       =     0.5000
redata: DM Tolerance for SCF             =     0.000100
redata: Require (free) Energy convergence in SCF =     F
redata: DM (free)Energy tolerance for SCF =     0.000010 eV
redata: Require Harris convergence for SCF =     F
redata: DM Harris energy tolerance for SCF =     0.000010 eV
redata: Using Saved Data (generic)   =     F
redata: Use continuation files for DM    =     F
redata: Neglect nonoverlap interactions  =     F
redata: Method of Calculation            =     Diagonalization
redata: Divide and Conquer               =     T
redata: Electronic Temperature           =     0.0019  Ry
redata: Fix the spin of the system       =     F
redata: Dynamics option                  =     Single-point calculation
redata: ***********************************************************************
Total number of electrons:    10.000000
Total ionic charge:    10.000000
 Kpoints in:            1 . Kpoints trimmed:            1

siesta: k-grid: Number of k-points =     1
siesta: k-grid: Cutoff (effective) =     2.555 Ang
siesta: k-grid: Supercell and displacements
siesta: k-grid:    1   0   0      0.000
siesta: k-grid:    0   1   0      0.000
siesta: k-grid:    0   0   1      0.000

                     ====================================
                        Single-point calculation
                     ====================================

outcell: Unit cell vectors (Ang):
        6.337723    0.000000    0.000000
        0.000000    5.109156    0.000000
        0.000000    0.000000    5.109156

outcell: Cell vector modules (Ang)   :    6.337723    5.109156    5.109156
outcell: Cell angles (23,13,12) (deg):     90.0000     90.0000     90.0000
outcell: Cell volume (Ang**3)        :    165.4366
New_DM. Step:     1
Initializing Density Matrix...
New grid distribution:   1
           1       1:   20    1:   16    1:   16

InitMesh: MESH =    40 x    32 x    32 =       40960
InitMesh: (bp) =    20 x    16 x    16 =        5120
InitMesh: Mesh cutoff (required, used) =   100.000   108.418 Ry
ExtMesh (bp) on 0 =    52 x    48 x    48 =      119808
PhiOnMesh: Number of (b)points on node 0 =                 5120
PhiOnMesh: nlist on node 0 =                37463

stepf: Fermi-Dirac step function

siesta: Program's energy decomposition (eV):
siesta: Ebs     =      -144.114838
siesta: Eions   =       832.688440
siesta: Ena     =       133.196299
siesta: Ekin    =       351.293331
siesta: Enl     =       -49.166483
siesta: DEna    =        -0.000000
siesta: DUscf   =         0.000000
siesta: DUext   =         0.000000
siesta: Exc     =      -152.746893
siesta: eta*DQ  =         0.000000
siesta: Emadel  =         0.000000
siesta: Emeta   =         0.000000
siesta: Emolmec =         0.000000
siesta: Ekinion =         0.000000
siesta: Eharris =      -556.129236
siesta: Etot    =      -550.112186
siesta: FreeEng =      -550.112186

   scf: iscf   Eharris(eV)      E_KS(eV)   FreeEng(eV)    dDmax  Ef(eV)
   scf:    1     -556.1292     -550.1122     -550.1122  0.86575 -7.9560
timer: Routine,Calls,Time,% =    IterSCF     1       0.024   2.99
   scf:    2     -555.8909     -555.8296     -555.8296  0.09239 -7.3982
   scf:    3     -555.8813     -555.8434     -555.8434  0.05858 -7.4409
   scf:    4     -555.8774     -555.8523     -555.8523  0.03731 -7.4659
   scf:    5     -555.8758     -555.8583     -555.8583  0.02388 -7.4805
   scf:    6     -555.8751     -555.8626     -555.8626  0.01535 -7.4890
   scf:    7     -555.8748     -555.8657     -555.8657  0.00992 -7.4940
   scf:    8     -555.8747     -555.8679     -555.8679  0.00645 -7.4970
   scf:    9     -555.8746     -555.8696     -555.8696  0.00421 -7.4987
   scf:   10     -555.8746     -555.8709     -555.8709  0.00277 -7.4998
   scf:   11     -555.8746     -555.8718     -555.8718  0.00183 -7.5004
   scf:   12     -555.8746     -555.8725     -555.8725  0.00122 -7.5008
   scf:   13     -555.8746     -555.8730     -555.8730  0.00082 -7.5011
   scf:   14     -555.8746     -555.8734     -555.8734  0.00056 -7.5012
   scf:   15     -555.8746     -555.8737     -555.8737  0.00038 -7.5013
   scf:   16     -555.8746     -555.8739     -555.8739  0.00026 -7.5014
   scf:   17     -555.8746     -555.8741     -555.8741  0.00018 -7.5014
   scf:   18     -555.8746     -555.8742     -555.8742  0.00013 -7.5015
   scf:   19     -555.8746     -555.8743     -555.8743  0.00009 -7.5015

SCF Convergence by dMax criterion
max |DM_out - DM_in|:     0.00008892
SCF cycle converged after   19 iterations

Using DM_out to compute the final energy and forces

siesta: E_KS(eV) =             -555.8746

siesta: E_KS - E_eggbox =      -555.8746

siesta: Atomic forces (eV/Ang):
----------------------------------------
   Tot    0.000000    0.001570   -0.000000
----------------------------------------
   Max    0.014455
   Res    0.008358    sqrt( Sum f_i^2 / 3N )
----------------------------------------
   Max    0.014455    constrained

Stress-tensor-Voigt (kbar):        3.99       -2.95       -2.95       -0.00        0.00       -0.00
(Free)E + p*V (eV/cell)     -555.8087
Target enthalpy (eV/cell)     -555.8746
Writing WFSX for COOP/COHP in n2.fullBZ.WFSX

siesta: Program's energy decomposition (eV):
siesta: Ebs     =      -139.544760
siesta: Eions   =       832.688440
siesta: Ena     =       133.196299
siesta: Ekin    =       359.286773
siesta: Enl     =       -61.601204
siesta: DEna    =        -0.743518
siesta: DUscf   =         0.460310
siesta: DUext   =         0.000000
siesta: Exc     =      -153.784825
siesta: eta*DQ  =         0.000000
siesta: Emadel  =         0.000000
siesta: Emeta   =         0.000000
siesta: Emolmec =         0.000000
siesta: Ekinion =         0.000000
siesta: Eharris =      -555.874605
siesta: Etot    =      -555.874605
siesta: FreeEng =      -555.874605

siesta: Final energy (eV):
siesta:  Band Struct. =    -139.544760
siesta:       Kinetic =     359.286773
siesta:       Hartree =     408.550992
siesta:    Ext. field =       0.000000
siesta:   Exch.-corr. =    -153.784825
siesta:  Ion-electron =   -1139.924821
siesta:       Ion-ion =     -30.002724
siesta:       Ekinion =       0.000000
siesta:         Total =    -555.874605

siesta: Stress tensor (static) (eV/Ang**3):
siesta:     0.002491   -0.000000    0.000000
siesta:    -0.000000   -0.001843    0.000000
siesta:    -0.000000    0.000000   -0.001843

siesta: Cell volume =        165.436585 Ang**3

siesta: Pressure (static):
siesta:                Solid            Molecule  Units
siesta:           0.00000434          0.00000398  Ry/Bohr**3
siesta:           0.00039836          0.00036583  eV/Ang**3
siesta:           0.63824503          0.58612805  kBar
(Free)E+ p_basis*V_orbitals  =        -555.456212
(Free)Eharris+ p_basis*V_orbitals  =        -555.456212

siesta: Electric dipole (a.u.)  =   -0.000000    0.000000    0.000000
siesta: Electric dipole (Debye) =   -0.000000    0.000001    0.000000

timer: Elapsed wall time (sec) =       1.187
timer: CPU execution times (sec):

Routine            Calls   Time/call    Tot.time        %
siesta                 1       2.612       2.612   100.00
Setup                  1       0.160       0.160     6.13
bands                  1       0.000       0.000     0.00
KSV_init               1       0.000       0.000     0.00
IterGeom               1       2.368       2.368    90.66
state_init             1       0.112       0.112     4.29
hsparse                1       0.000       0.000     0.00
overlap                1       0.108       0.108     4.13
Setup_H0               1       0.508       0.508    19.45
naefs                  2       0.002       0.004     0.15
MolMec                 2       0.000       0.000     0.00
kinefsm                2       0.054       0.108     4.13
nlefsm                 2       0.178       0.356    13.63
DHSCF_Init             1       0.048       0.048     1.84
DHSCF1                 1       0.004       0.004     0.15
INITMESH               1       0.000       0.000     0.00
DHSCF2                 1       0.044       0.044     1.68
REMESH                 1       0.004       0.004     0.15
REORD                 56       0.000       0.004     0.15
PHION                  1       0.032       0.032     1.23
COMM_BSC              44       0.000       0.004     0.15
POISON                22       0.009       0.196     7.50
fft                   44       0.003       0.148     5.67
IterSCF               19       0.076       1.444    55.28
setup_H               19       0.073       1.380    52.83
DHSCF                 21       0.080       1.672    64.01
DHSCF3                21       0.071       1.492    57.12
rhoofd                21       0.027       0.560    21.44
cellXC                21       0.016       0.336    12.86
vmat                  20       0.013       0.268    10.26
writeHSX              19       0.002       0.032     1.23
compute_dm            19       0.002       0.044     1.68
diagon                19       0.002       0.044     1.68
r-eigvec              20       0.002       0.044     1.68
r-buildHS             20       0.000       0.000     0.00
rdiag                 20       0.002       0.044     1.68
rdiag1                20       0.001       0.016     0.61
rdiag2                20       0.000       0.004     0.15
rdiag3                20       0.001       0.024     0.92
rdiag4                20       0.000       0.000     0.00
r-buildD              19       0.000       0.000     0.00
MIXER                 18       0.000       0.000     0.00
WriteDM               19       0.000       0.000     0.00
PostSCF                1       0.268       0.268    10.26
DHSCF4                 1       0.180       0.180     6.89
dfscf                  1       0.160       0.160     6.13
overfsm                1       0.004       0.004     0.15
state_analysis         1       0.036       0.036     1.38
siesta_move            1       0.000       0.000     0.00
writewave              1       0.000       0.000     0.00
optical                1       0.000       0.000     0.00
  
>> End of run:   5-DEC-2017  10:12:56
Job completed
